examples/serverless/main.tf

```diff
@@ -130,6 +130,57 @@ resource "aws_rds_cluster_parameter_group" "example_mysql" {
   tags        = local.tags
 }
 
+################################################################################
+# RDS Aurora Module - MySQL Serverless V2
+################################################################################
+
+module "aurora_mysql_serverlessv2" {
+  source            = "../../"
+  name              = "${local.name}-mysqlv2"
+  engine            = "aurora-mysql"
+  engine_mode       = "provisioned"
```

Comment:
actually after testing this, looks like we can't use the data source since its returning multiple matches and I'm not able to come up with a filter pattern to scope it down to one